### ISSUE TITLE
Hadronox fight bugs with big bugs

### ISSUE THREAD

**Issue by mooers** [dalaranwow/dalaran-wow#5324]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
que'd for a dungeon, im lvl 72 priest; i think its the crypts ? the one with hadranox for the first boss; it was me, Ladavole, Taenel, Delessaria and Ceboose as the random group. I joined in and as we got the flash across your screen "hadranox makes its way up the tunnel", we pulled the first 2 large beetle like guards, a third spawned and we clearered them. Everything was fine except those big beetle elites that are down there to start the hadranox fight, they kept spawning on top of each other.  It wasnt immediate, it took maybe 30seconds until another would pop up full hp in its place. It got the point where 4 were stacked where one should be. So as you can imagine, us trying to fight through these guys - and hadronox below making his way up the tunnel- we suddenly had 8 elites on the group instead of 2 or 3 at a time. It kept resetting and repeating; the respawn trigger or time for the big beetle guys wasnt stopping. I died inbetween so I'm not sure if when the boss reset, all of the beetles did too because we just left. I left and then of course got dungeon deserter :( 
**Expected behaviour**: Tell us what should happen instead
I believe in retail, they respawn but not at that rate and definitely not before the previous two had already died. There were multiple respawns of the same monster on top of each other- to the point where when you pulled one, it actually was 4 or 3 stacked ontop and then, wipe. 
**Steps to reproduce the problem**:

1. 
2. 
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Lothloryen** [comment]
Hello, closing this for now. If anyone experiences it update this issue, please.

Thank you.
